lineprotocol/decoder_test.go

```diff
@@ -494,8 +550,8 @@ func TestDecoderSkipSection(t *testing.T) {
 func removeTestSeparators(s string) string {
```

Comment:
does this just need to be removed entirely now?

Reply:
yup! good call, done.